sample-kotlin-model/src/main/kotlin/com/vimeo/dummy/sample_kotlin/KotlinConcreteExample.kt

```diff
@@ -12,4 +12,22 @@ class KotlinConcreteExample : KotlinGenericExample<String>() {
 
     var kotlinObject: KotlinSamples? = null
 
+    override fun equals(other: Any?): Boolean {
```

Comment:
Are `sample-kotlin-model` and `sample-java-model` library only related modules or they are examples for the end user?

Reply:
They are primarily used as test cases, but they also serve as examples. In the future I will be moving all the "samples" into an integration test module, and will leave the sample project as just a sample.